pkg/bundle/resources.go

```diff
@@ -150,23 +147,9 @@ func chartURL(location *fleet.HelmOptions, auth auth) (string, error) {
 	return repoURL.ResolveReference(chartURL).String(), nil
 }
 
-func addCharts(directories []directory, base string, charts []*fleet.HelmOptions, client *client.Getter) ([]directory, error) {
+func addCharts(directories []directory, base string, charts []*fleet.HelmOptions, auth Auth) ([]directory, error) {
 	for _, chart := range charts {
 		if _, err := os.Stat(filepath.Join(base, chart.Chart)); os.IsNotExist(err) || chart.Repo != "" {
-			auth := auth{}
-			if chart.SecretName != "" {
-				c, err := client.Get()
-				if err != nil {
-					return nil, err
-				}
-				secret, err := c.Core.Secret().Get(client.Namespace, chart.SecretName, metav1.GetOptions{})
-				if err != nil {
-					return nil, err
-				}
-				auth.username = string(secret.Data[corev1.BasicAuthUsernameKey])
-				auth.password = string(secret.Data[corev1.BasicAuthPasswordKey])
-			}
-
```

Comment:
Should we keep this as a flag for existing users?

Reply:
@nickgerace Yeah, I change the design. Now these are flags `--username-file` `--password-file` and `--cabundle-file`